mmdet3d/core/bbox/structures/coord_3d_mode.py

```diff
@@ -237,11 +239,13 @@ def convert_point(point, src, dst, rt_mat=None):
             else:
                 rt_mat = rt_mat @ rt_mat.new_tensor([[1, 0, 0], [0, 0, 1],
                                                      [0, -1, 0]])
-        elif src == Coord3DMode.LIDAR and dst == Coord3DMode.DEPTH:
+        elif (src == Coord3DMode.LIDAR
```

Comment:
How about keeping the `rt_mat is None` in an inner if branch like the code in `box_3d_mode.py`? This alternative seems easier to understand since the if logic is clearer.